app/models/manageiq/providers/embedded_ansible/seeding.rb

```diff
@@ -0,0 +1,61 @@
+module ManageIQ::Providers::EmbeddedAnsible::Seeding
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def seed
+      provider = ManageIQ::Providers::EmbeddedAnsible::Provider.in_my_region.first_or_initialize
+      provider.update!(
+        :name => "Embedded Ansible"
+      )
+
+      manager = provider.automation_manager || provider.build_automation_manager
+      manager.update!(
+        :name => "Embedded Ansible",
+        :zone => MiqServer.my_server.zone # TODO: Do we even need zone?
+      )
+
+      manager.authentications.create_with(
+        :type => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::MachineCredential"
+      ).find_or_create_by!(
+        :name => "#{Vmdb::Appliance.PRODUCT_NAME} Default Credential"
+      )
+
+      create_local_playbook_repo
+      local_repo = ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource.find_or_create_by!(
+        :name       => "#{Vmdb::Appliance.PRODUCT_NAME} Default Project",
+        :manager_id => manager.id
+      )
+      local_repo.raw_update_in_provider(
+        :scm_type             => "git",
+        :scm_url              => "file://#{local_playbook_repo_dir}",
+        :scm_update_on_launch => false
+      )
+    end
+
+    private
+
+    def create_local_playbook_repo
+      Ansible::Content.consolidate_plugin_playbooks
```

Comment:
If you're still looking for names... mishmash, mingle, hodgepodge, amalgam, merge